base/test_views.py

```diff
@@ -1,8 +1,11 @@
-from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
 from splinter import Browser
 
 
-class TestBaseViews(TestCase):
+test_url = 'http://localhost:8081/'
```

Comment:
There is no need to write the test_url  as a global variable, as it is already an attribute of StaticLiveServerTestCase.

According to Django documentation
> By default the live server listens on localhost and picks the first available port in the 8081-8179 range. Its full URL can be accessed with self.live_server_url during the tests.